docs/api/response.md

```diff
@@ -263,6 +263,36 @@ Hello wörld!
 > });
 > ```
 
+## HTTP Redirects
+
+To redirect incoming HTTP requests to a new location you can define your HTTP
+response like this:
+
+```php
+<?php
+
+// …
+
+$app->get('/blog.html', function () {
+    return new React\Http\Message\Response(
+        React\Http\Message\Response::STATUS_FOUND,
+        [
+            'Location' => '/blog'
+        ]
+    );
+});
+```
+
+Redirect responses have a status code in the 3xx range, and a [`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)
+response header holding the URL to redirect to. When a Browser or a search engine
+crawler receives a redirect, they will automatically follow the new URL provided
+in the `Location` header.
+
+See [response status codes](#status-codes) for more details.
```

Comment:
Definitely a useful link, but perhaps we can make this more useful by providing more context, e.g. a list of common status codes for redirects?

Reply:
I added a list with commonly used redirect status codes, WDYT?